build/bootstrap/bootstrap-debian.sh

```diff
@@ -70,3 +72,25 @@ sudo mv /tmp/bazelisk /usr/bin/bazel
 
 # Install the Unison file-syncer.
 . bootstrap/bootstrap-unison.sh
+
+# Configure environment variables for CockroachDB and Managed Service.
+echo 'export PATH="${PATH}:$HOME/go/src/github.com/cockroachdb/cockroach/bin:/usr/local/go/bin"' >> ~/.bashrc_bootstrap
```

Comment:
It seems you should only be doing this stuff if you are cloning the repo, right? If you don't clone the repo, there's no benefit to updating the `PATH` like this.

Reply:
That's a valid point. I kept it there because seemed harmless at first. I'll move this.